php/Blockonomics.php

```diff
@@ -538,6 +538,19 @@ public function is_payment_recorded($status, $order, $wc_order){
         return true;
     }
 
+    public function is_transaction_recorded($status, $order, $wc_order){
+        if (!metadata_exists('post',$wc_order->get_id(),'blockonomics_'. $order['crypto'] .'_txid', $order['txid']) )  {
+            return false;
+        }
+        return true;
+    }
```

Comment:
*is_transaction_recorded* and *record_transaction* can be merged into a single function. 
It is not a particularly good idea to keep having very small functions (unless they are used multiple times) as it leads to unnecessary function calls and code complexity

Reply:
Renamed and refactored in 9e0e8b2